### PR TITLE
Mixee sample: corrections, add Kotlin tab

### PR DESCRIPTION
**Pull request by deining**

### CODE REVIEW

File: docs/index.adoc [hunk 1]
```diff
@@ -6443,6 +6451,109 @@ class Sub implements Runnable {
...
+import org.apache.logging.log4j.core.config.Configurator
+
+import picocli.CommandLine
+import picocli.CommandLine.Command
+import picocli.CommandLine.Mixin
+import picocli.CommandLine.Model.CommandSpec
+import picocli.CommandLine.Option
+import picocli.CommandLine.ParseResult
+import picocli.CommandLine.RunLast
+import picocli.CommandLine.Spec
```
Comment (remkop) [inline docs/index.adoc:6469]: I would prefer to reduce the imports somehow; maybe use star imports? 

```
import picocli.CommandLine.*; //?
```

Too many imports can distract from the main point of the example...

File: docs/index.adoc [hunk 2]
```diff
@@ -6443,6 +6451,109 @@ class Sub implements Runnable {
...
+        // Each subcommand that mixes in the LoggingMixin has its own instance
+        // of this class, so there may be many LoggingMixin instances.
+        // We want to store the verbosity value in a single, central place,
+        // so we find the top-level command,
+        // and store the verbosity level on our top-level command's LoggingMixin.
+        (mixee.root().userObject() as MyApp).loggingMixin.verbosity = verbosity
+    }
+}
+
+@CommandLine.Command(name = "app", subcommands = [Sub::class])
```
Comment (remkop) [inline docs/index.adoc:6495]: I prefer `@Command` to `@CommandLine.Command`

File: docs/index.adoc (hunk 1 above)
Comment (deining) [reply docs/index.adoc]: I fully agree.
The PR uses star imports now.

File: docs/index.adoc (hunk 2 above)
Comment (deining) [reply docs/index.adoc]: I overlooked that. I prefer @Command to @CommandLine.Command, too

**Comment by remkop** [comment]
Merged, and published the rendered HTML.
Thank you for the contribution!

**Pull request merged**